Dockerfile

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the Egeria project
+
+# Thes are optional tags used to add additional metadata into the docker image
+# These may be supplied by the pipeline in future - until then they will default
+
+ARG version=2.10-SNAPSHOT
+ARG EGERIA_BASE_IMAGE=odpi/egeria:latest
```

Comment:
How about having EGERIA_BASE_IMAGE=odpi/egeria -- so that a user override with their own repo if needed, but removing the latest tag as this overlaps effectively with 'version' above? (and is probably useful to keep seperate?)